src/app/Main/Footer/Footer-LinkList.component.tsx

```diff
@@ -41,7 +41,9 @@ const linkStyle = css`
     }
 `;
 
-export function FooterLinkListComponent(): JSX.Element {
+export function FooterLinkListComponent(props: {
+    supportMail: string | undefined;
```

Comment:
Reject: What happens if supportMail is undefined? Perhaps you could check it in the container and display an error message or disable the link here and show a tooltip with an error message. Or fetch the mail in the main container with the other stuff, use the error mechanism there and put the serverInfo into a context object used by the footer container.
You can also add a ticket if this would be to time consuming and it was not defined in the "support mail from api" ticket.

Reply:
I think it is not the right way not to show the complete page just because the SupportMail is not delivered by the server (error mechanism of the QueryPage). This seems to be overkill. Blocking the link seems more appropriate to me.

Reply:
Done. Commit "Handle SupportMail is undefined"

Reply:
Yes, that's a difficult decision, how user friendly or error tolerant the app should be. For the mibi-portal i came to the solution to show a loading spinner blocking the whole page until all necessary data is fetched. But that's not implemented yet, the mibi-portal for now just looks normal, even if the api is not working. That was some kind of a problem during a release, we thought all was fine but then users complained because the api was not running...